### ISSUE TITLE
issue with sequence where the parts have different durations and easing

### ISSUE THREAD

**Issue by kschaefers** [julianshapiro/velocity#442]
I am using velocity since yesterday so I am still new to it.
I am trying to make a person walk around a block. The walking is not the problem, but the duration. I have created a sequence and I am starting it directly afterwards:

    var walkingAroundBlock = [
        {e: $('.walkingAroundBlock'), p: {top: '842px', left: '1036px'}, o: {duration: 1, easing: "linear", before: function (el) {
                    $(el).css('background-position', '0 0')
                }, complete: function (el) {
                    $(el).css('z-index', '16');
                }}},
        {e: $('.walkingAroundBlock'), p: {top: '890px', left: '952px'}, o: {duration: 1600, easing: "linear", complete: function (el) {
                    $(el).css('z-index', '15').css('background-position', '100% 0');
                }}},
        {e: $('.walkingAroundBlock'), p: {top: '790px', left: '784px'}, o: {duration: 3200, easing: "linear", complete: function (el) {
                    $(el).css('z-index', '14').css('background-position', '33.33% 0');
                }}},
        {e: $('.walkingAroundBlock'), p: {top: '696px', left: '952px'}, o: {duration: 3200, easing: "linear", complete: function (el) {
                    $(el).css('z-index', '14').css('background-position', '66.66% 0');
                }}},
        {e: $('.walkingAroundBlock'), p: {top: '794px', left: '1120px'}, o: {duration: 3200, easing: "linear", complete: function (el) {
                    $(el).css('z-index', '16').css('background-position', '0 0');
                }}},
        {e: $('.walkingAroundBlock'), p: {top: '842px', left: '1036px'}, o: {duration: 1600, easing: "linear", complete: function (el) {
                    $(el).css('z-index', '16');
                    $.Velocity.RunSequence(walkingAroundBlock);
                }}}
    ];

    $.Velocity.RunSequence(walkingAroundBlock);

The person now moves around the block, but ignores the duration and the easing of the first parts. It moves with the default duration of 400 and "swing" easing. Only the last part is not ignored and the person now needs 1600ms to walk the last part and the walking is "linear".
As soon as I remove the last part and only let the person walk to the last corner of the block it has the same effect and only the last part has an effect.
Because in the end there will be several moving elements with different values in my city, I can not just change the default duration. This would mean I would constantly have to change the default duration and easing.

I hope you can help me with this problem

I have uploaded a reduced version of my small application here: http://kschaefers.de/velocityProblem/

**Comment by julianshapiro** [comment]
1) nothing shows up at your link
2) make sure you're using the latest version of velocity and the ui pack. a fix was pushed recently that fixed this sort of odd behavior.

if that doesn't fix your problem, let me know and i'll re-open this issue so we can figure it out :-D ty!

**Comment by kschaefers** [comment]
I don't know what is wrong with the link, it works just fine for me. But well, I guess it does not matter any more.
I made sure, that I really have the most recent version and I guess that fixed it. I was on version 1.2.1 instead of 1.2.2 and version 5.0.3 instead of 5.0.4 for the ui pack.
I just downloaded velocity and the ui pack this week, so I guess I downloaded directly before the new version came out.
Sorry for the trouble - problem has been fixed for me with the newest versions.
